src/main/java/sysc4806/project24/mini_survey_monkey/controllers/RespondController.java

```diff
@@ -21,7 +24,26 @@ public RespondController(SurveyRepository surveyRepository) {
     public String respond(@PathVariable("surveyID") int surveyID, Model model) {
         Survey survey = surveyRepository.findById(surveyID);
         model.addAttribute("survey", survey);
+        // TODO: Handle form having no questions
```

Comment:
maybe adding if statement to check if there is no question then we redirect to a page where tell user no question has been added

Reply:
Do you think it's okay if I add this in a future PR?

Reply:
Yes definitely